apps/chat/src/store/prompts/prompts.epics.ts

```diff
@@ -708,11 +708,15 @@ const importPromptsEpic: AppEpic = (action$) =>
             },
           );
 
+          const emptyFolders = promptsHistory.folders.filter((folder) =>
```

Comment:
Why it's named `empty`?

Reply:
This is name for the folders without prompts inside.